src/runtime/runtime.ts

```diff
@@ -29,6 +29,7 @@ export type RuntimeArcOptions = Readonly<{
   stub?: boolean;
   listenerClasses?: ArcInspectorFactory[];
   inspectorFactory?: ArcInspectorFactory;
+  context?: Manifest;
```

Comment:
When we talked offline, we agreed that this is a wrong approach, so this PR is blocked by https://github.com/PolymerLabs/arcs/pull/3151

Reply:
done - removed this.